packages/generator-langium/package.json

```diff
@@ -38,7 +38,9 @@
   },
   "devDependencies": {
     "@types/lodash": "^4.14.170",
-    "@types/yeoman-generator": "^5.0.0"
+    "@types/yeoman-generator": "^5.0.0",
+    "@types/yeoman-test": "^4.0.3",
+    "yeoman-test": "^7.0.0"
```

Comment:
Please add a `"test": "vitest run"` script so we can run this directly from the package folder